triaging.rst

```diff
@@ -3,193 +3,243 @@
 Triaging an Issue
 =================
 
-When you have the Developer role on the `issue tracker`_ you are able to triage
-issues directly without any assistance.
+This section of the devguide documents the `issue tracker`_ for users
+and developers. 
 
-Fields
-------
+Contributors with the Developer role on the `issue tracker`_ can triage issues
+directly without any assistance.
+
+Fields in the Issue Tracker
+---------------------------
+
+The major elements found in an issue report include:
+
+* Classification (including *Title*) - These fields categorize the issue
+  The fields include *Title*, *Type*, *Stage*, *Components*, and *Version*.
+* Process - These fields indicate the state of the issue and its progress
+  toward resolution. The fields are *Status*, *Resolution*, *Dependencies*,
+  *Superseder*, *Assigned To*, *Nosy List*, *Priority*, *Keywords*, *Comment*,
+  *File*, *File Description*, *Remote hg repo*, *GitHub PR*.
+* Messages
+* History
 
 Title
 '''''
-Should be properly descriptive of what the issue is about. Occasionally
-people file an issue that either has too generic of a title or end up thinking
-they filed about X but in fact it turns out to be about Y and thus the
-title is now wrong.
+A brief description of the issue. Review whether the title is too generic or
+specifies an incorrect term or library.
```

Comment:
I mentioned this in my other comment, but mentioning it here too.  This may need to be a new issue for further discussion.  I think it would be helpful for the title to begin with the module that the issue is for.  A large portion of the tickets have this now, but not all of them.  It makes search (and sorting) easier to find tickets related to a module.

Reply:
Let's recommend that folks do as you suggest. Do you have a format that has worked well?

Reply:
Thanks for asking!  :-)  Well, for IDLE, we use `IDLE:` and I think that let's everyone know they can skip it.  :-)  Seriously though, if you go into the bug tracker and then search, then click 'Sort on' next to title, you'll see (with sort descending), that the `zlib` and `zip` modules come up first.  That's really just the format I'd suggest, putting the module first instead of somewhere else in the title.  I don't think a colon after it is necessary, unless it reads better.  I've downloaded all the bugs using the .csv download and realized this could make it a lot easier for people to find related tickets.  The old 'I think there's a ticket for this, but I can't find it' becomes easier when you can narrow the search.  :-)  It also gives a sense for the modules that may be neglected (for example, argparse has a ton of open tickets).

Reply:
@csabella Added below and in the checklist.